README.md

```diff
@@ -0,0 +1,46 @@
+# Honeybadger for Crystal
+
+HTTP::Handler and exception notifier for the [⚡Honeybadger error notifier](https://www.honeybadger.io/).
+
+## Getting Started
+
+Update your `shard.yml` to include honeybadger:
+
+```diff
+dependencies:
++  honeybadger:
++    github: honeybadger-io/honeybadger-crystal
+```
+
+Add the Honeybadger::Handler to the HTTP::Server stack:
+
+```crystal
+honeybadger_api_key = ENV["HONEYBADGER_API_KEY"]? || "00000000"
+honeybadger_enabled = MyServer.production?
+
+Honeybadger.configure(api_key: honeybadger_api_key)
+
+Honeybadger::Handler.new(
+  enabled: honeybadger_enabled,
+  factory: Honeybadger::Payload
```

Comment:
Could `factory` default to `Honeybadger::Payload` so that you don't have to provide it here by default? Also, could `enabled` default to `true` for the same reason? It would be simpler if all you had to do was this:

server = HTTP::Server.new([
   HTTP::LogHandler.new(Log.for("http.server")),
   HTTP::ErrorHandler.new,
   Honeybadger::Handler.new,
   router
 ])

Reply:
Certainly, yep. My instinct is that the payload will almost never be the bare http::payload -- it's not equipped to provide context like controller or actions, nor user_id, etc. It'll always be something like `Honeybadger::LuckyPayload` etc.